piety-promoted-v3/modernized/17-appendix.adoc

```diff
@@ -274,6 +282,9 @@ JONAS BINNS, a boy in his fifteenth year, son of Jonathan Binns, of Crawshawboot
 in Lancashire, departed this life the 23rd of the Ninth month, 1760.
 His surviving father gave a testimony concerning him, which I nearly copy.
 
+[.embedded-content-document.testimony]
```

Comment:
Not sure if this should have been embedded either.

Reply:
Yeah, I think this one could go either way, but I prefer it embedded like you did it 👍